example/reader_util.c

```diff
@@ -121,9 +121,9 @@ void ndpi_analyze_payload(struct ndpi_flow_info *flow,
 			  u_int8_t *payload,
 			  u_int16_t payload_len,
 			  u_int32_t packet_id) {
```

Comment:
Maybe print a warning or error if allocation fails, because there is no return value that marks a failure.

Reply:
Make sense. I'll do

Reply:
Done in latest revision